CGFloatType.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |spec|
   spec.name = 'CGFloatType'
-  spec.version = '1.0.2'
+  spec.version = '1.0.3'
```

Comment:
Thanks for the pull request. But in future please don't change the version here in a pull request unrelated to making a release.

These changes would result in a minor release not a patch release due to API changes.